library/src/main/java/com/google/android/exoplayer2/extractor/mp3/Mp3Extractor.java

```diff
@@ -52,6 +57,19 @@
   };
 
   /**
+   * Flags controlling the behavior of the extractor.
+   */
+  @Retention(RetentionPolicy.SOURCE)
+  @IntDef(flag = true, value = {FLAG_ENABLE_CONSTANT_BITRATE_SEEKING})
+  public @interface Flags {}
+
+  /**
+   * Flag to enable using the {@link ConstantBitrateSeeker} for cases such as Xing Header present
```

Comment:
Flag to force enable seeking using a constant bitrate assumption in cases where seeking would otherwise not be possible.

Note: You shouldn't @link to ConstantBitrateSeeker because it's package private, so there's no page generated to link to when the Javadoc is actually generated (we only generate Javadoc for public and protected visibilities).